core/src/main/java/org/primefaces/extensions/component/timer/Timer.java

```diff
@@ -345,6 +346,14 @@ public void setForward(final boolean forward) {
         getStateHelper().put(PropertyKeys.forward, forward);
     }
 
```

Comment:
Also look at how Calculator does `locale`...
```java
public Locale calculateLocale() {
        if (appropriateLocale == null) {
            final FacesContext fc = FacesContext.getCurrentInstance();
            appropriateLocale = LocaleUtils.resolveLocale(fc, getLocale(), getClientId(fc));
        }
        return appropriateLocale;
    }
```
LocaleUtils is smart and will use Locale if defined or the global faces-config.xml locale or the browsers locale.